sharing_configs/admin.py

```diff
@@ -170,19 +168,27 @@ def import_from_view(self, request, **kwargs):
             self.model._meta.app_label,
             self.model._meta.model_name,
         )
+        main_url = f"admin:{info[0]}_{info[1]}_import"
+        ajax_url = f"admin:{info[0]}_{info[1]}_ajax"
+        context = Context()
+        context.update({"main_url": main_url, "ajax_url": ajax_url})
         if request.method == "POST":
             form = self.get_sharing_configs_import_form(request.POST)
             if form.is_valid():
                 folder = form.cleaned_data.get("folder")
                 filename = form.cleaned_data.get("file_name")
                 obj = SharingConfigsClient()
                 try:
-                    resp_api_dict = obj.import_data(folder, filename)
+                    resp_api_bytes = obj.import_data(folder, filename)
+                    content = base64.b64encode(resp_api_bytes)
+                    obj = self.get_sharing_configs_import_data(content)
                     msg = format_html(
-                        _("The (file) object has been imported successfully!"),
+                        _(
+                            f"The (file) object {str(obj)} has been imported successfully!"
```

Comment:
```suggestion
                            f"The object {obj} has been imported successfully!"
```
f-string already uses `str()` under the hood